assets/src/edit-story/app/media/media3p/useFetchMediaEffect.js

```diff
@@ -27,12 +27,15 @@ import { useEffect } from 'react';
  */
 import { __ } from '@wordpress/i18n';
 import { useSnackbar } from '../../snackbar';
-import { ProviderType } from '../../../components/library/panes/media/common/providerType';
+import { ProviderType } from '../providerType';
+import usePrevious from '../usePrevious';
 import { useMedia3pApi } from './api';
 
 function getFetchMediaErrorMessage(provider) {
   if (provider === ProviderType.UNSPLASH) {
     return __('Error loading media from Unsplash', 'web-stories');
+  } else if (provider === ProviderType.COVERR) {
+    return __('Error loading media from Coverr', 'web-stories');
```

Comment:
The display names ('Unsplash' or 'Coverr') should be read from the provider configuration.

Reply:
I couldn't find a way of doing i18n message interpolation using __ (seems very basic), so I've moved the messages to the config.